posthog/api/test/test_event.py

```diff
@@ -61,9 +61,7 @@ def test_filter_events(self):
         _create_event(event="$pageview", team=self.team, distinct_id="some-random-uid", properties={"$ip": "8.8.8.8"})
         _create_event(event="$pageview", team=self.team, distinct_id="some-other-one", properties={"$ip": "8.8.8.8"})
 
-        expected_queries = (
```

Comment:
A tuple with only one element must end with a comma otherwise Python won't know it's a tuple. In this case I don't think we wanted to use a tuple.